.github/workflows/codespell.yml

```diff
@@ -6,8 +6,8 @@ on:
     - main
 
   pull_request:
-    branches:
-    - main
+    #branches:
```

Comment:
The reason I had modfied this is because it would'nt trigger workflows if they didn't target the main branch. I think it would be useful to run the tests on all PRs, even if they target branches other than `main`

Reply:
Got it, that makes sense. We can remove the branch section entirely then or yes just leave it commented out.

Reply:
Either way is fine. I just commented them out here, because I wanted to see what tests are working/failing on this PR. We can change workflows also in a separate PR and remove this change here before merging if you prefer.